src/plugins/utils/LambdaEventUtils.ts

```diff
@@ -360,6 +363,32 @@ class LambdaEventUtils {
         return className;
     }
 
+    static injectTriggerTagsForNetlify(span: ThundraSpan, pluginContext: any, originalContext: any): string {
+        const className = ClassNames.NETLIFY;
+        const domainName = DomainNames.API;
+        let operationName = 'netlify-site';
+
+        const siteName = process.env[NetlifyConstants.NETLIFY_SITE_NAME];
+
+        if (siteName) {
+            operationName = siteName;
+        }
+
+        const originalHandler = process.env[EnvVariableKeys._HANDLER];
+        const functionName = operationName + '/' + originalHandler.substring(0, originalHandler.indexOf('.'));
```

Comment:
I think, it is better to be at safe side here by checking whether original handler is available as env var and it has `.` character